### DISCUSSION TITLE
Move crates_io_* crates into `crates/` dir

### DISCUSSION THREAD

**Discussion by 0xPoe** [rust-lang/crates.io#8255] [Feature Requests]
Currently, we have numerous files in the crates.io project's root. I suggest moving these individual crates into a sub-directory to reduce the length of the file list in the IDE's explorer. 

If it is acceptable I am willing to update it.

**Comment by Turbo87** [comment]
yeah, I had similar thoughts lately. I guess the main `crates_io` package would need to stay top-level (unless we adjust the Heroku buildpack accordingly), but the other can probably be moved. The smoke test workflow might also have to be adjusted for the new location.

**Reply by 0xPoe** [reply]
Thanks for your reply! I will give it a shot.
